### ISSUE TITLE
Can't Read '.cfg' and Documentation MIA

### ISSUE THREAD

**Issue by cerniuk** [ryanmaxwell/UncrustifyX#16]
Had this working in the spring as was thrilled it existed. Used some code generators and batch-fixed their output, best thing since sliced bread (almost).

Reloaded now and there is no functionality. (?!?)  Download has no dictionary and no default config.  Worse yet, located my old default config ".cfg" and it just spews errors in UncrustifyX.

![uncrustifyx](https://cloud.githubusercontent.com/assets/369530/8513229/cb1b43b0-232f-11e5-9945-6e130994a140.png)

And the documentation is empty:
![documentation](https://cloud.githubusercontent.com/assets/369530/8513233/e2c2b624-232f-11e5-882f-8caffdc6e1af.png)

But what is especially perplexing is the litany of errors from trying to read in one of it's previous ".cfg" files or any other found on the web:
![errors_in_reading_cfg_file](https://cloud.githubusercontent.com/assets/369530/8513186/5bf49ff0-232e-11e5-9f5d-264b8a6f846e.png)

Here are the example '.cfg' files that used to work ...
https://cerniuk.org/wiki/find?keyword=uncrustify

So I gave the compile a try.  The external dependences were the death of this attempt.  The tool from Rentzsch gave me pause but ... ok.  Loaded that and got past one build error. Would be nice to not have to depend on an external tool... 

mogenerator
Wolf Rentzsch
https://github.com/rentzsch/mogenerator

Then got to th Fragaria tool...  
Fragaria
Jonathan Mitchell
https://github.com/mugginsoft/Fragaria

Turns out it is not ARC and not smoothly converted.  I tried a simple convert but after getting a litany of errors, it occurred to me that this tool may want other tools to be built... and so on and so on...

Long story short, without UncrustifyX, BBUncrustifyPlugin-Xcode by Benoît Bourdon becomes less attractive as figuring out exactly what the uncrustify settings do at the conf field level is a long tedious process of trial and error.  Great companion tool, I miss UncrustifyX!

**Comment by ryanmaxwell** [comment]
@cerniuk Sorry that it's not working for you. The code hasn't had an update for quite some time - And yes, Fragaria was extremely useful but it's non-arc (I think it's even GC - now fully deprecated) so it's likely time to find a replacement/update.

As for your empty windows - try trashing the app data (~/Library/Application Support/UncrustifyX) and re-opening the app.

**Comment by cerniuk** [comment]
Trashing the app data didn't work.  Spent over 4 hours hand working the uncrustify settings manually to get them to work with bbuncrustify plugin. Now I really miss your app!  Thanks for prior and thanks for trying.
